examples/grpc-client/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"io"
+	"log"
+	"os"
+
+	"github.com/pkg/errors"
+	runnerv1 "github.com/stateful/runme/internal/gen/proto/go/runme/runner/v1"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var (
+	addr       = flag.String("addr", "127.0.0.1:7890", "the address to connect to")
+	file       = flag.String("file", "", "file with content to upper case")
+	resultFile = flag.String("write-result", "-", "path to a result file (default: stdout)")
+)
+
+func main() {
+	flag.Parse()
+
+	if err := run(); err != nil {
+		log.Fatalf("error: %v", err)
+	}
+}
+
+func run() error {
+	conn, err := grpc.Dial(*addr, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		return errors.Wrap(err, "failed to connect")
+	}
+	defer conn.Close()
+
+	client := runnerv1.NewRunnerServiceClient(conn)
+
+	stream, err := client.Execute(context.Background())
```

Comment:
```suggestion
	g, ctx := errgroup.WithContext(context.Background())

	stream, err := client.Execute(ctx)
```

Stream needs to close early if there's an error, for example, reading the file